maven/templates/deploy.py

```diff
@@ -54,6 +68,30 @@ def md5(fn):
     return hashlib.md5(open(fn).read()).hexdigest()
 
 
+def update_pom_within_jar(jar_path, new_pom_content):
+    ext = 'jar'
+    original_jar_basename = os.path.basename(jar_path[:-len(ext) - 1])
+    updated_jar_basename = original_jar_basename + '-updated'
```

Comment:
@lolski make sure we keep the old names in final version

Reply:
Good point, addressed in a911aea04576669bfeae464158eb457613f1f20a